source/arriving/setup_computer.rst

```diff
@@ -1,51 +1,167 @@
 
-VPN
-    Follow `these instructions <https://www.usherbrooke.ca/services-informatiques/repertoire/reseaux/rpv/>`_ to connect through **VPN**.
+.. code-block:: bash
 
-    **(we might want to write a summary for English speakers)**
+    git config --global user.name "FIRST_NAME LAST_NAME"
+    git config --global user.email "MY_NAME@example.com"
 
-SSH
-    1. Connect to the University's VPN.
+You can also add these lines in your .bashrc (:bash:`code ~/.bashrc`) to see the current Git branch in your terminal:
 
-    2. You must know your lab computer's IP address or its University code (ex: DINF-0000-00a). (Ask casius at https://casius.usherbrooke.ca/sp if you don't know).
+.. code-block:: bash
 
-    3. | Connect with ssh. On Linux or Mac, ssh can be simply used via the terminal. The option -X is to make sure the applications you use appear at home.
-       | ``ssh -x your_cip@your_computer_IPaddress``, or
-       | ``ssh -X your_cip@DINF-0000-00a.dinf.fsci.usherbrooke.ca``.
+    # Custom prompt for Git 
+    git_branch() {
+    git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/[\1] /'
+    }
+    PS1="\$(git_branch)$PS1"
+
+In the next steps, you will have to *clone* and *fork* Git repositories. In short, *cloning* a repository means "downloading" it to your computer using the :bash:`git clone` command with the URL found by clicking on the green "<> Code" button on the Github page of the repository. Moreover, *forking* a repository means creating your own version of the repository by clicking on the "Fork" button on the Github page of the repository.
+
+More details on the use of Git are available here: :ref:`ref_git`.
+
+Scilpy
+""""""
+
+`Scilpy <https://github.com/scilus/scilpy>`_ is the main library supporting research and development at the lab. It currently supports python versions 3.8 to 3.10, so make sure you have followed all the previous steps. Once your python distribution is correctly installed, Scilpy can be installed by following the procedure outlined below:
+
+    .. code-block:: bash
+
+        # If you never installed libfreetype6-dev
+        sudo apt install libfreetype6-dev
+
+        # If you intend to use COMMIT/AMICO (on LINUX or WSL)
+        sudo apt install libblas-dev liblapack-dev
+
+        # If you intend to use COMMIT/AMICO (on MAC)
+        brew install openblas lapack
+
+        # Make a fork of scilpy to be able to modify your own version of the code.
+        # Go where you want the scilpy folder to be, then:
+        git clone https://github.com/YOUR_USERNAME/scilpy.git # Don't forget to replace YOUR_USERNAME
```

Comment:
un screenshot de github? and here we show https we will need to update that

Reply:
I explain in the Git section what fork and clone mean.

Reply:
I updated for the SSH version and added a paragraph on SSH key in the Git section. This paragraph could become more explicit in the future (I just refer to the Github page, which is pretty weel done).